src/helper.ts

```diff
@@ -489,6 +522,11 @@ export function addOptionsSoldAction(event: OptionsSold): void {
   const ethPrice = chainlinkAggregatorContract.latestAnswer()
   optionsSoldAction.ethPrice = ethPrice
 
+  const collateralAmount = getCollateralDepositedFromLogs(seller, txLogs)
+
+  optionsSoldAction.isOpen = collateralAmount !== '0'
+  optionsSoldAction.collateralAmount = BigInt.fromString(collateralAmount)
+
```

Comment:
Given the checks and the values being stored, makes more sense to return a BigInt from the function.